### ISSUE TITLE
FFMPEG: tag 0x58564944/'DIVX' 

### ISSUE THREAD

**Issue by 22itay** [ShirAmir/K9andFriends#1]
I get the following warning

OpenCV: FFMPEG: tag 0x58564944/'DIVX' is not supported with codec id 13 and format 'mp4 / MP4 (MPEG-4 Part 14)'
OpenCV: FFMPEG: fallback to use tag 0x00000020/' ???'

**Comment by ShirAmir** [comment]
yes it is due to opencv's usage of videos I had not really gotten into it.
The program still works though
